composer.json

```diff
@@ -16,12 +16,11 @@
       }
     },
     "require": {
-        "dxw/rubbishthorclone": "dev-master",
+        "dxw/rubbishthorclone": "^1.0",
```

Comment:
Should we set the minimum version as `1.0.1` to ensure we get the one that uses `v1.0.0` of option parser?

Reply:
`composer` will complain if it can't install a stable version. And it'll always use the most recent version unless it has a good reason not to.